### ISSUE TITLE
.net 4.7.1 compatability.

### ISSUE THREAD

**Issue by RogerMostad** [scriptcs/scriptcs#1290]
I have been having a problem with the current Scriptcs.  I need to recompile it under 4.7.1.  Is this been attempted?  I am getting a tone of issues on the tests.  

I am constantly getting an error loading the Assembly 'System.Runtime' it is reporting a couldn't find file.

**Comment by filipw** [comment]
are you trying to compile manually in VS or are you running the build scripts? There should definitely be no issues in running scriptcs under 4.7.1. However the source at the moment targets 4.6.1 (which was driven by netstandard2.0), so you will need 4.6.1 to compile.

**Comment by RogerMostad** [comment]
Sorry for the delay yes this was a build issue.  I think the transition away from the packages.json for Nuget face planeted on my system.  I regenerated my projects from scratch.

We decided to ditch the attempt to use a third party library that was just released at the 4.7.1 level.  Who creates a libarary that supports over a decade of hardware and release it for only the latest framework!?

Sorry I should have got back here  earlier.  

This issue can be closed as far as I am concerned.

Thanks again for all of contributers time and effort.
